meson.build

```diff
@@ -0,0 +1,36 @@
+project('timeshift', 'vala', 'c')
+
+dependencies = [
+    dependency('glib-2.0'),
+    dependency('gobject-2.0'),
+    dependency('gtk+-3.0'),
+    dependency('gio-2.0'),
+    dependency('gio-unix-2.0'),
+    dependency('json-glib-1.0'),
+    dependency('vte-2.91'),
+    dependency('gee-0.8'),
+    dependency('xapp'),
+    meson.get_compiler('vala').find_library('posix'),
+    meson.get_compiler('c').find_library('m'),
+]
+
+am_cflags = [
+    '-fstack-protector',
+    '--std=c99',
```

Comment:
Set this in the project's `default_options: ['c_std=c99']`.

Reply:
Fixed